includes/reader-activation/class-reader-activation.php

```diff
@@ -1959,7 +1959,7 @@ public static function register_reader( $email, $display_name = '', $authenticat
 
 		$user_id = false;
 
-		if ( $existing_user ) {
+		if ( $existing_user && self::is_reader_without_password( $existing_user ) ) {
```

Comment:
We don't want to send magic links to accounts with passwords so the auth form doesn't load into the OTP state for these the next time they attempt to login